contracts/bonding/BondingManager.sol

```diff
@@ -593,8 +598,10 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     function pendingFees(address _delegator, uint256 _endRound) public view returns (uint256) {
         uint256 currentRound = roundsManager().currentRound();
         Delegator storage del = delegators[_delegator];
+
         // End round must be before or equal to current round and after lastClaimRound
```

Comment:
Let's remove this comment since the revert reasons below already explain what the require statements do.